handbook/engineering/README.md

```diff
@@ -751,13 +751,32 @@ When this occurs, we will begin receiving the following error message when attem
 
 5. Accept the new terms of service.
 
-
 ## Responsibilities
 
-TODO
-
 > work in progress, contributions welcome, please just make only one small change at a time per PR.  See https://fleetdm.com/handbook/company/leadership#vision-for-dept-handbook-pages for info
 
+### Renew MDM CSR Certificate
```

Comment:
1. re: "Certificate" https://fleetdm.com/handbook/company/why-this-way#why-does-fleet-use-sentence-case
2. Except... actually, since CSR == certificate signing request, shouldn't it just be "CSR"?
3. Let's expand the expand the acronym:  e.g. "Certificate signing request (CSR)"

Reply:
Not 100% sure I got what you were looking for but there was an attempt. Please confirm :D